mailpoet/lib/Twig/Filters.php

```diff
@@ -20,6 +21,16 @@ public function getFilters() {
         'replaceLinkTags',
         'MailPoet\Util\Helpers::replaceLinkTags'
       ),
+      new TwigFilter(
+        'wpKses',
+        [$this, 'wpKses'],
+        ['is_safe' => ['html']]
+      ),
     ];
   }
+
+  public function wpKses($content, $allowedHtml) {
+    $wp = WPFunctions::get();
+    return $wp->wpKses($content, $allowedHtml);
+  }
```

Comment:
@costasovo, I considered this approach but I opted to sacrifice the developer experience in favor of being more specific in the list of allowed HTML tags. I'm not sure if this is the best approach or not in this case. My fear is that, down the road, we might end up wanting to add more HTML tags to the list of allowed HTML that would apply to all calls of this filter. Instead of just one specific call where it might be needed. Some of those need tags could be potentially problematic.

As an alternative to having to set the list of allowed HTML tags, developers can call the filter with one of the context names like `post` supported by `wp_kses()`. This should work in a way very similar to what you are suggesting. Would this be enough in your opinion, or would you still prefer that we implement a list of allowed HTML tags in the filter?

Reply:
@rodrigoprimo Ok. Makes sense. Let's keep it as is. Maybe in the future, we will add more, and it will be easier to decide what is the best approach. We could also define our own context or contexts.

Reply:
Sounds good to me. Thanks.